docs/setup-sensor/podman.md

```diff
@@ -0,0 +1,153 @@
+---
+title: Installing Orb using Podman Quadlet
+shortTitle: Podman Quadlet
+metaDescription: Run the Orb sensor as a Podman container managed by a systemd quadlet on any compatible system.
+section: setup-sensor
+layout: guides
+imageUrl: ../../images/devices/podman.png
+subtitle: 'Difficulty: Intermediate 🧑‍🔬'
+---
+
+# Installing Orb using Podman Quadlet
+
+## Introduction
+
+[Podman](https://podman.io/) is a daemonless container engine that runs containers in isolated environments. This guide shows you how to run the Orb sensor as a Podman container managed by a systemd quadlet, a declarative way to define and manage containers as system services. This method is ideal for Linux systems where systemd is available, such as servers, firewalls, routers, or personal computers with Podman installed.
+
+This guide assumes you have Podman already installed and running on your host system. All commands assume being ran as the root user.
+
+## Prerequisites
+
+Before you begin, make sure you have:
+
+- A Linux host machine with Podman installed (version 4.0 or later for quadlet support). See the official [Podman installation guides](https://podman.io/getting-started/installation).
+  - Install on Fedora: `dnf install podman`
+  - Install on Ubuntu: `apt-get install podman`
+- Systemd installed and running (standard on most Linux distributions).
+- Basic familiarity with using the command line or terminal on your host system.
+- Your host machine must be connected to the network you wish to monitor.
+- Rootful Podman, as the Orb sensor requires `CAP_NET_RAW` for network monitoring.
+
+Verify Podman installation:
+
+```bash
+podman --version
+```
+
+## Quick install
+
+If you'd like to run Orb with auto-updates and don't want to get into the details, run these commands:
+
+```bash
+mkdir -p /etc/containers/systemd
+curl -fsSL https://orb.net/docs/scripts/podman/orb-sensor.container -o /etc/containers/systemd/orb-sensor.container
+systemctl daemon-reload
+systemctl start orb-sensor
+systemctl enable orb-sensor
```

Comment:
`systemctl enable orb-sensor` returns an error
```
Failed to enable unit: Unit /run/systemd/generator/orb-sensor.service is transient or generated.
```
From reading into it a bit, it seems normal - the service is enabled automatically, and orb will start on reboot
> It is not possible to enable a generated service yes, because they are in general not persistent identifiers. In the case of quadlet, the generator itself handles enabling though, so if the *.container file has an [Install] section, the correct links will be generated to automatically start the service.
> In other words, it is like the enable is run automatically.

I think that line can just be removed.